### ISSUE TITLE
java.net.UnknownHostException: central.maven.org

### ISSUE THREAD

**Issue by Zwilcox96** [DependencyTrack/dependency-track#594]
I installed the executable war to try it out and I am running into this error a lot whenever I upload a BOM. I can't seem to resolve central.maven.org from any of my devices or external services. 

OS Name:      Linux
OS Version:   2.6.32-754.25.1.el6.x86_64
OS Arch:      amd64
CPU Cores:    4
Java Vendor:  Oracle Corporation
Java Version: 8.0.232
Java Home:    /usr/lib/jvm/java-1.8.0-openjdk-1.8.0.232.b09-1.el6_10.x86_64/jre
Application:  Dependency-Track
Version:      3.7.1
Built-on:     2020-01-07T07:00:06Z

Framework:    Alpine
Version :     1.7.0
Built-on:     2020-01-07T06:01:27Z

```
15:27:36.479 ERROR [MavenMetaAnalyzer] Request failure
java.net.UnknownHostException: central.maven.org
	at java.net.InetAddress.getAllByName0(InetAddress.java:1281)
	at java.net.InetAddress.getAllByName(InetAddress.java:1193)
	at java.net.InetAddress.getAllByName(InetAddress.java:1127)
	at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:45)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:112)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:374)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:393)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:186)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
	at org.dependencytrack.tasks.repositories.MavenMetaAnalyzer.analyze(MavenMetaAnalyzer.java:80)
	at org.dependencytrack.tasks.repositories.RepositoryMetaAnalyzerTask.analyze(RepositoryMetaAnalyzerTask.java:87)
	at org.dependencytrack.tasks.repositories.RepositoryMetaAnalyzerTask.inform(RepositoryMetaAnalyzerTask.java:52)
	at alpine.event.framework.BaseEventService.lambda$publish$0(BaseEventService.java:99)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by stevespringett** [comment]
Please execute https://github.com/DependencyTrack/dependency-track/blob/master/upgrade/3.7.1_and_previous_maven_central_fix.sql against your database. This has been fixed in master (v3.8) but is not being backported to 3.7.x

**Comment by davidkarlsen** [comment]
Will upgrade to 3.8 apply fine with the fix added manually onto a 3.7.1?

**Comment by stevespringett** [comment]
Yes @davidkarlsen. You can safely apply the fix for 3.7.1 today, and the upgrade to 3.8 will still work fine.
